addons/product/views/product_views.xml

```diff
@@ -82,7 +82,7 @@
                                 </div>
                                 <group name="email_template_and_project" invisible="1"/>
                             </group>
-                            <group string="Description for Customers" name="description">
+                            <group string="Description for Customers" name="description" invisible="1">
```

Comment:
Why is it invisible ?

Reply:
The "description for customers" is currently (saas-11.5) not visible if you only install the website_sale module.
I made this field invisible to make it only visible in sale_management to comply with that behavior.

Now that I think about it, it does not make sense that this description is not available if you only install the website_sale. Since it is shown to customers on the product's "shop" page.

Maybe we should change this? What do you think @tivisse ?